layouts/partials/site-header.html

```diff
@@ -26,11 +26,11 @@ <h1 class="font-extrabold text-3xl md:text-5xl *:block">{{ . | markdownify }}</h
                   <p class="mt-6 text-sm md:text-xl">{{ $headertextmore | markdownify }}{{ end }}</p>
                   {{ end }}
 
-                 <!--  {{ if .Params.headeractions }}
-                  <div class="flex my-8 justify-center sm:justify-start lg:mt-12">
+                 {{ if .Params.headeractions }}
+                  <div class="flex gap-x-2 justify-center items-center sm:justify-start mt-8 lg:mt-12">
                     {{ range .Params.headeractions }}
                     {{ if in .action "Download" }}
-                    <button id="{{ .id }}" class="text-xs md:text-base text-center font-semibold rounded-lg px-5 md:px-2 xl:px-12 py-3 lg:py-5 bg-[#192E5B] hover:bg-[#72A3C0] uppercase mr-3">
+                    <button id="{{ .id }}" data-dropdown-toggle="dropdown" class="text-xs md:text-base text-center font-semibold rounded-lg px-5 md:px-2 xl:px-12 py-3 lg:py-5 bg-[#192E5B] hover:bg-[#72A3C0] uppercase mr-3">
```

Comment:
By adding `data-dropdown-toggle` we're now using Flowbite to handle the dropdown menu functionality.